packages/backend/src/main/java/com/keyVault/app/dto/PasswordDTO.java

```diff
@@ -0,0 +1,74 @@
+package com.keyVault.app.dto;
+import java.util.Date;
+import com.keyVault.app.entity.Category;
+import com.keyVault.app.entity.Icon;
+public class PasswordDTO {
+	private int id;
+	private String content;
+	private Date createdAt;
+	private int seenqty;//number of times password seen
```

Comment:
```suggestion
	private int seenQty;//number of times password seen
```

its a short way to say seen quantity